11ty/definitions/pt_PT/capacitismo.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Capacitismo / Aptidismo
+slug: capacitismo
+defined: true
+speech: noun
+flag:
+  level: warning
+  text: content warning
+reading:
+- text: "Capacitismo (wikipedia)"
+  href: https://pt.wikipedia.org/wiki/Capacitismo
+---
+
+Um sistema que valoriza o corpo e a mente das pessoas com base em ideias construídas socialmente de normalidade, inteligência, excelência e produtividade. Essas ideias construídas estão profundamente enraizadas no anti-negritude, na eugenia, no colonialismo e no capitalismo.
```

Comment:
"no anti-negritude" should be "na anti-negritude". "Negritude" is a feminine noun in portuguese.: https://dicionario.priberam.org/negritude

Reply:
Well caught